src/tests/timesync/TimeSync_Synchronize_001.test.js

```diff
@@ -0,0 +1,37 @@
+import { pluginDeactivate, pluginActivate, syncTime } from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'TimeSync - Synchronize 001',
+  description: 'Check the Synchronize Functionality of TimeSync Module',
+  steps: [
+    {
+      description: 'Check if TimeSync Plugin is stopped correctly',
+      test: pluginDeactivate,
+      params: constants.timeSyncPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Check if Time Sync Plugin is started correctly',
+      test: pluginActivate,
+      params: constants.timeSyncPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Invoke Time Synchronize',
+      sleep: 5,
+      test() {
+        return syncTime.call(this)
+      },
+      validate(res) {
+        //TODO - Implement Proper validation to check whether the time Synchronization is completed
```

Comment:
You can do the following:
- Set time to somethign different, like 2 days in the past or in advance
- Call synchronise
- Check if time is back to curren time (as on previous test)